scripts/latte/sample_dp.py

```diff
@@ -0,0 +1,262 @@
+# Adapted from Latte
```

Comment:
we can enable this feature by default. so maybe no need to have a new file

Reply:
Enabled in sample.py